README.md

```diff
@@ -175,34 +175,42 @@ reporting_delay <- estimate_delay(
 ```
 
 If data was not available we could instead make an informed estimate of
-the likely delay (*this is a synthetic example and not applicable to
-real world use cases and we have not included uncertainty to decrease
-runtimes*),
+the likely delay. We choose a lognormal distribution with mean 2,
+standard deviation 1 (both with some uncertainty) and a maximum of 10.
+*This is just an example and unlikely to apply in any particular use
+case*.
 
 ``` r
 reporting_delay <- dist_spec(
-  mean = convert_to_logmean(2, 1), sd = convert_to_logsd(2, 1), max = 10,
+  mean = convert_to_logmean(2, 1),
+  mean_sd = 1,
+  sd = convert_to_logsd(2, 1),
+  sd_sd = 1,
+  max = 10,
   dist = "lognormal"
 )
 #> Warning: The meaning of the 'max' argument has changed compared to previous versions. It now indicates the maximum of a distribution rather than the length of the probability mass function (including 0) that it represented previously. To replicate previous behaviour reduce max by 1.
 #> This warning is displayed once every 8 hours.
+reporting_delay
+#> 
+#>   Uncertain lognormal distribution with (untruncated) logmean 0.58 (SD 1) and logSD 0.47 (SD 1)
```

Comment:
This is quite a lot of uncertainty given the mean + quite a high maximum delay given how short the delay is. Potentially causing the issues?

Reply:
Ah I see you got this in https://github.com/epiforecasts/EpiNow2/pull/481/commits/fe3d17ab0b698f4b24081edc033ad086eee94ce7